tests/bud.bats

```diff
@@ -4473,15 +4475,15 @@ EOM
   _prefetch alpine
   _prefetch debian
 
-  run_buildah build --build-arg base=alpine --build-arg toolchainname=busybox --build-arg destinationpath=/tmp --pull=false $WITH_POLICY_JSON -f $BUDFILES/from-with-arg/Containerfile .
+  run_buildah build --build-arg base=alpine --build-arg toolchainname=busybox --build-arg destinationpath=/tmp --pull=ifmissing $WITH_POLICY_JSON -f $BUDFILES/from-with-arg/Containerfile .
```

Comment:
This one shouldn't need changing - the only referenced images are prefetched.  Does it start to fail without this change?

Reply:
It pulls busybox I believe.

Reply:
Well-spotted.  If it's not using the "debian" image, might as well change it to prefetch busybox instead.